models/project/project.go

```diff
@@ -240,6 +241,27 @@ func (p *Project) GetDefaultBranch() (string, error) {
 	return group[1], nil
 }
 
+// Returns cached default branch if exists.
```

Comment:
```suggestion
// GetCachedDefaultBranch returns cached default branch if exists.
```

> Comments should begin with the name of the thing being described and end in a period:

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences